### PR TITLE
Cleanup the platform check scripts which have been moved

### PR DESCRIPTION
**Pull request by wangxin**
### Description of PR

Summary:
Fixes # (issue)

### Type of change

- [x] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

#### What is the motivation for this PR?
The "check_*.py" scripts under platform_tests have been re-organized to
common/platform/*_utils.py. These files are kind of duplicated and should be cleaned up.

#### How did you do it?
This commit is to cleanup these check scripts and update the dependent test scripts to use the modules under common/platform.

#### How did you verify/test it?
Test run all the affected test scripts. No import issue is found.

### CODE REVIEW

File: tests/platform_tests/test_reboot.py [hunk 1]
```diff
@@ -68,7 +68,7 @@ def check_interfaces_and_services(dut, interfaces, reboot_type = None):
     @param interfaces: DUT's interfaces defined by minigraph
     """
     logging.info("Wait until all critical services are fully started")
-    check_critical_services(dut)
+    wait_critical_processes(dut)
```
Comment (neethajohn) [inline tests/platform_tests/test_reboot.py:71]: Is this change intentional? replacing 'check_critical_services' with 'wait_critical_processes'. I see that both functions serve different purposes. 

Same question for all changes related to these 2 functions

Comment (wangxin) [reply tests/platform_tests/test_reboot.py:71]: Yes. The function was renamed in the `tests.common.platform.process_utils.py` module. But their purposes are the same.

Comment (neethajohn) [reply tests/platform_tests/test_reboot.py:71]: I see a check_critical_processes also in that file. Its purpose is different from 'wait'. It is to ensure that the processes continue to remain up for a certain duration

Comment (wangxin) [reply tests/platform_tests/test_reboot.py:71]: The `check_critical_processes` function watches the critical services for specified time. Once any of the critical service is not running during the watch period, the script will fail by `pytest_assert`. Just after reboot, not all critical services are running. If we use `check_critical_processes`, the test will always fail unnecessarily.

Comment (neethajohn) [reply tests/platform_tests/test_reboot.py:71]: I agree that after reboot, it makes sense to use 'wait' in line 60. but in the teardown module, line 42 should be 'check' because we need to ensure that the processes are still up.

Comment (wangxin) [reply tests/platform_tests/test_reboot.py:71]: Ah, I see, that's a good suggestion! Thanks! I'll updte.

**Comment by wangxin** [comment]
retest this please
